workers/unity/Packages/com.improbable.gdk.core/Logging/ILogDispatcher.cs

```diff
@@ -1,10 +1,9 @@
-using System;
 using Improbable.Worker.Core;
 using UnityEngine;
 
 namespace Improbable.Gdk.Core
 {
-    public interface ILogDispatcher : IDisposable
```

Comment:
This was added by Firtina for the cleanup on some MonoBehaviour tests to work, in this commit: https://github.com/spatialos/UnityGDK/commit/f9bdfdb85404fc0d1856b203dec72d7ea3e52198

So I think the merge should have the MonoBehaviour branch version being applied.

Reply:
Interesting. THe tests are passing even without this change. Do you remember the context behind this change? Is this to ensure that Dispose() is called on ForwardingDispatchers?

Reply:
Reverted changes. ILogDispatcher is an IDisposable again.